lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -328,13 +328,15 @@ def _monitor(self, poll_interval):
 
     def _poll_stats(self, datapath, cookie: int = 0, cookie_mask: int = 0):
         """
-        Send a FlowStatsRequest message to the datapath
+        Send a FlowStatsRequest message to the datapath and
+        remove old enforcement rules
         Raises:
         MagmaOFError: if we can't poll datapath stats
         """
         try:
             flows.send_stats_request(datapath, self.tbl_num,
                                      cookie, cookie_mask)
+            self.delete_all_flows(datapath)
```

Comment:
sorry I meant your api, get_stats. move that there.
and not delete_all_flows but self._delete_old_flows(records)